src/WebViewTypes.ts

```diff
@@ -646,6 +655,16 @@ export interface IOSWebViewProps extends WebViewSharedProps {
    */
   onContentProcessDidTerminate?: (event: WebViewTerminatedEvent) => void;
 
+  /**
+   * Function that is invoked when the `WebView` should open a new window.
+   * 
+   * This happens when the JS calls `window.open('http://someurl', '_blank')`
+   * or when the user clicks on a `<a href="http://someurl" target="_blank"` link.
+   *
+   * @platform ios
+   */
+  onOpenWindow?: (event: WebViewOpenWindowEvent) => void;
```

Comment:
Can you define this in `WebViewSharedProps` instead of having duplicate versions in `IOSWebViewProps`/`AndroidWebViewProps`?

Reply:
Are you sure? `WebViewSharedProps` would also impact `MacOSWebViewProps` and `WindowsWebViewProps`. But I did not implement this feature on those platforms.

Reply:
Hm. My thinking was that's what the `@platform` annotations were for (for example, `javascriptEnabled`... although I'm now seeing that one is kind of a lie because it also works on iOS... hm...).

But then there are some properties like `userAgent` that are duplicated. So I guess there's not a strict standard here and I'm okay with this if you prefer it.

Reply:
The only documentation I found about `@platform` is https://reactnative.dev/docs/platform-specific-code

Seems like this is more of a documentation convention by react-native but not a type guard?

Reply:
You're correct, but I'm not sure that the separate interfaces provides that much of a type guard either, since they wind up getting merged with

```typescript
export type WebViewProps = IOSWebViewProps & AndroidWebViewProps & WindowsWebViewProps;
```

But I'm okay leaving this.